src/components/Footer.tsx

```diff
@@ -0,0 +1,123 @@
+import classNames from 'classnames';
+
+import { useContext, useEffect, useState } from 'react';
+import { Todo } from '../types/Todo';
+import { deleteTodos } from '../api/todos';
+import { AuthContext } from './Auth/AuthContext';
+
+type Props = {
+  todos: Todo[];
+  setVisibleTodos: (arg: Todo[]) => void;
+  visibleTodos: Todo[];
+  setIsCompletedTodosDeleting :React.Dispatch<React.SetStateAction<boolean>>,
```

Comment:
```suggestion
  setVisibleTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
  visibleTodos: Todo[];
  setIsCompletedTodosDeleting: React.Dispatch<React.SetStateAction<boolean>>;
```
Please pay attention to your code style. Sometimes you use semicolons, sometimes commas. And keep one style through your app.